src/main.cpp

```diff
@@ -756,7 +756,7 @@ void loop() {
         orientationReading.z = radians(orientationData.orientation.z);
 
         for (u_int8_t n = 0; n < 4; n++) {
-            lightSensors[n] = ads1115.readADC_SingleEnded(n);
+            lightSensors[n] = 0; //ads1115.readADC_SingleEnded(n);
```

Comment:
This needs removing (probably being shown because at the moment this branch is in conflict with master, and this is one of the conflicting blocks)

Reply:
exactly, it needs updating now PR#19/issue 13 (https://github.com/robberwick/tauradigm-teensy/pull/19) was merged in